sonic-chassisd/scripts/chassisd

```diff
@@ -163,6 +170,9 @@ class ModuleUpdater(logger.Logger):
                                CHASSIS_MODULE_INFO_SLOT_FIELD,
                                CHASSIS_MODULE_INFO_OPERSTATUS_FIELD]
 
+        chassis_state_db = daemon_base.db_connect("CHASSIS_STATE_DB")
```

Comment:
This new table is kept in CHASSIS_STATE_DB, unlike the other tables like MIDPLANE_TABLE etc that is kept in STATE_DB ?

Reply:
See this comment in PR description that the info is kept in CHASSIS_STATE_DB because it's accessible by swss containers. Does the swss use this asic present info ?

Reply:
There are a couple of places we can keep asic info like STATE_DB (redis instance) or CHASSIS_STATE_DB (redis chassis instance). 

CHASSIS_STATE_DB is accessible from every where in swss like`/usr/bin/swss.sh or /usr/bin/local/swss.sh` - script creating swss container` and `docker_init.sh` inside swss container. Inside swss container, STATE_DB is not accessible (asic database is mapped, instead of database).  So CHASSIS_STATE_DB is a safer choice.